src/containers/Pages/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchAllJobs, jobsSelector } from "../../store/slices/JobsSlice";
+import JobList from "./components/JobList/JobList";
+
+function JobContainer() {
+  const dispatch = useDispatch();
+  dispatch(fetchAllJobs());
```

Comment:
Why do we need to fetch jobs here?
I see we are fetching jobs in JobList

Also, fetching jobs should be placed in useEffect, if not, it will always fetch when JobContainer re-render